packages/desktop-client/src/components/mobile/transactions/TransactionList.tsx

```diff
@@ -429,7 +460,10 @@ function SelectedTransactionsFloatingActionBar({ transactions, style }) {
             style={{ width: 200 }}
           >
             <Menu
-              getItemStyle={getMenuItemStyle}
+              getItemStyle={item => ({
+                ...getMenuItemStyle(),
+                ...(item.name === 'delete' && { color: theme.errorTextMenu }),
+              })}
```

Comment:
The menu in L348 does not have a `delete` `MenuItem`, and typescript was complaining it referencing a value which it has no overlap.
![image](https://github.com/user-attachments/assets/ec53e1f7-6ebd-4afd-9922-79794ef70e8a)
So I am extracting that conditional out as an inline function.